examples/js/loaders/MMDLoader.js

```diff
@@ -1283,16 +1320,12 @@ THREE.MMDLoader = ( function () {
 
 			if ( this.tgaLoader === null ) {
 
-				if ( THREE.TGALoader === undefined ) {
-
-					throw new Error( 'THREE.MMDLoader: Import THREE.TGALoader' );
-
-				}
-
-				this.tgaLoader = new THREE.TGALoader( this.manager );
+				throw new Error( 'THREE.MMDLoader: Please inject an instance of THREE.TGALoader via .setTGALoader()' );
 
 			}
 
+			this.tgaLoader.manager = this.manager;
```

Comment:
Is this `manager` overridden necessary? Personally I prefer doing in user side unless there's any special reasons doing in the loader here. 

```javascript
var loader = new THREE.MMDLoader(manager).setTGALoader(new THREE.TGALoader(manager));
```

Some users might want to set different manager to loaders.

Reply:
Internally loaders used to be configured this way. It might be also necessary to set the path of  `TGALoader` like in `ColladaLoader` or `FBXLoader` which overwrites user defined values. Hence, I personally would prefer if just the instance is set without any configurations.

Reply:
>  It might be also necessary to set the path of TGALoader like in ColladaLoader or FBXLoader which overwrites user defined values. 

Sorry, I couldn't get (maybe because of my bad English). Could you elaborate for me please?

At least I think we should add note about manager set to TGALoader will be overridden with the one of  MMDLoader.

Reply:
Check out this line for example:

https://github.com/mrdoob/three.js/pull/16771/files#diff-844b1cc1e411ca1bec3c6b4115a2af4cL3894

Reply:
I understand what you meant. Not MMDLoader but other loaders can update path of TGALoader. (And maybe MMDLoader should do, too.)

Regarding, overriding manager, honestly I'm still on the fence. But yeah maybe sounds a reasonable solution. Thanks for updating the doc.

Reply:
Yeah this seems debatable to me as well... Most people are using the default loading manager anyway, so this only becomes relevant when the user is overriding the manager. And if those users _have_ chosen to use a different manager for TGALoader, shouldn't we preserve that?

Reply:
Okay okay, then let's make the code more safe^^. How about overwriting `TGALoader.manager` only if the default loading manager is set? I've updated the PR with this approach.